sdk/ml/azure-ai-ml/tests/test_configs/pipeline_jobs/serverless_compute/serverless_pipeline_default_compute_valid/pipeline_component.yml

```diff
@@ -0,0 +1,24 @@
+$schema: https://azuremlschemas.azureedge.net/development/pipelineComponent.schema.json
+type: pipeline
+
+name: simple_pipeline_component
+
+jobs:
+  vanilla_node:
+    type: command
+    component: ../components/component.yml
+  node_with_resources:
+    type: command
+    component: ../components/component_with_resources.yml
+  node_with_compute_target:
+    type: command
+    component: ../components/component.yml
+  node_with_compute_target_and_resources:
```

Comment:
why do we say this is node_with_compute_target?

Reply:
just miss specify compute target for this node, will add this.

Reply:
Updated and added these missing compute target configurations.